Resources/Prototypes/Roles/Jobs/Medical/chief_medical_officer.yml

```diff
@@ -44,6 +44,7 @@
     components:
     - type: PsionicBonusChance #Nyano - Summary: makes it more likely to become psionic.
       flatBonus: 0.025
+    - type: CPRTraining
```

Comment:
Why is this not;
```suggestion
    special:
    - !type:AddComponentSpecial
      components:
      - type: CPRTraining

```

Reply:
like the other roles?

Reply:
It is

Reply:
:(

Reply:
Surprise, the fact that other roles use multiple instances of AddComponentSpecial is itself incorrect and unecessary. AddComponentSpecial accepts a list of components with arguments, there is no need to add more instances of this system. I simply needed to add the CPRTraining component to the list.